src/css/style.css

```diff
@@ -90,14 +90,6 @@ picture {
   display: block;
 }
 
-/* Inherit fonts for inputs and buttons */
-input,
-button,
-textarea,
-select {
-  font: inherit;
-}
-
```

Comment:
**NOTE: This change is only applicable to the media query breakpoint that you've used**

- Please, kindly ensure the min-width value you've used is not less than `768px`. As you used 
`@media screen and (min-width: 40em) {` which is equivalent to `640px`, Please kindly increase it to either `48em` or higher to meet the project requirement as this is a core requirement to have your project approved.

Reply:
As there is not any other media query other than mobile and desktop using 40em and 48 will not differ.
buy the way I will change it to 48em